package.json

```diff
@@ -8,6 +8,8 @@
     "console": "ts-node bin/console.ts",
     "format": "prettier --write \"src/**/*.ts\"",
     "start": "ts-node -r tsconfig-paths/register src/main.ts",
+    "db:test:reset": "(export NODE_ENV=test; sequelize db:create && sequelize db:drop && sequelize db:create && sequelize db:migrate)",
+    "db:reset": "sequelize db:create && sequelize db:drop && sequelize db:create && sequelize db:migrate && npm run db:test:reset",
```

Comment:
Alomejor vale: 
`sequelize db:seed:undo:all  `   =>  Deletes data from the database

Reply:
The command ```db:seed:undo:all``` only delete the seed user, not the users created in the e2e test